client/src/index.jsx

```diff
@@ -37,17 +37,17 @@ class ApateezGallery extends React.Component {
     this.getRequestWithId(id);
   }
   gotoHotNew() {
-    location.href = '/restaurants/' + 'ChIJA8_SN2eAhYARCIvEx44Zvfw';
+    location.href = '/restaurants/' + '2342';
   }
   gotoCitysBest() {
-    location.href = '/restaurants/' + 'ChIJUUjhfoaAhYARRuSNp1R18vs';
+    location.href = '/restaurants/' + '234';
   }
   searchRestaurant(searchValue) {
     $.ajax({
       url: BASE_URL + searchValue,
```

Comment:
Maybe switch to axios (to remove jquery entirely) and use relative paths until deploying proxies.